unifi/Dockerfile

```diff
@@ -1,7 +1,9 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:8.2.0
+ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:9.0.4
 # hadolint ignore=DL3006
 FROM ${BUILD_FROM}
 
+ARG BUILD_ARCH=amd64
+
 # Set shell
```

Comment:
```suggestion
# Set shell
ARG BUILD_ARCH=amd64
```

Reply:
I've squashed this to the `Install mongodb 4.4`, thanks!